src/Dashboard.tsx

```diff
@@ -74,32 +73,6 @@ function Dashboard() {
     await refetchUserProgress()
   }, [refetchUserProgress])
 
-  const {
-    data: latestActionRun
-  } = useGetUserUpdateTimeQuery(user?.id)
-
-  const formatTimeWithTimezone = useCallback((dateObj: Date) => {
-    const formatter = new Intl.DateTimeFormat('en-GB', {
-      day: '2-digit',
-      month: 'short',
-      year: 'numeric',
-      hour: '2-digit',
-      minute: '2-digit',
-      hour12: false,
-      timeZone: Intl.DateTimeFormat().resolvedOptions().timeZone
-    });
-
-    return formatter.format(dateObj)
-  }, [])
-
-  const expectedNextUpdateTime = useMemo(() => {
-    if (latestActionRun == null) {
-      return null
-    }
-    const expectedTime = new Date(latestActionRun.getTime() + 30 * 60 * 1000)
-    return expectedTime
-  }, [latestActionRun])
```

Comment:
Are we saying here that we're good with avoiding the sync/extending the duration between syncs? It is a constant factor that eats into our free minutes @damithc